webtask.json

```diff
@@ -21,6 +21,6 @@
   "auth0": {
     "createClient": true,
     "onUninstallPath": "/.extensions/on-uninstall",
-    "scopes": "read:client_grants create:client_grants delete:client_grants update:client_grants read:clients update:clients delete:clients create:clients read:client_keys update:client_keys delete:client_keys create:client_keys read:connections update:connections delete:connections create:connections read:resource_servers update:resource_servers delete:resource_servers create:resource_servers read:rules update:rules delete:rules create:rules read:rules_configs update:rules_configs delete:rules_configs read:email_provider update:email_provider delete:email_provider create:email_provider read:tenant_settings update:tenant_settings read:grants delete:grants read:guardian_factors update:guardian_factors read:email_templates create:email_templates update:email_templates read:roles update:roles delete:roles create:roles read:prompts update:prompts read:branding update:branding read:hooks update:hooks delete:hooks create:hooks"
+    "scopes": "read:client_grants create:client_grants delete:client_grants update:client_grants read:clients update:clients delete:clients create:clients read:client_keys update:client_keys delete:client_keys create:client_keys read:connections update:connections delete:connections create:connections read:resource_servers update:resource_servers delete:resource_servers create:resource_servers read:rules update:rules delete:rules create:rules read:rules_configs update:rules_configs delete:rules_configs read:email_provider update:email_provider delete:email_provider create:email_provider read:tenant_settings update:tenant_settings read:grants delete:grants read:guardian_factors update:guardian_factors read:mfa_policies update:mfa_policies read:email_templates create:email_templates update:email_templates read:roles update:roles delete:roles create:roles read:prompts update:prompts read:branding update:branding read:hooks update:hooks delete:hooks create:hooks"
```

Comment:
Question for reviewer: will tenants that already installed this extension get the new scopes, or they need to uninstall+reinstall?

Reply:
In this case, it will require an uninstall/reinstall so that the deploy CLI client is created with the new scopes. 

In other cases, if the changes are just code-related, then bumping the patch field of an extension will automatically roll out the new version of an extension to everyone who has it installed. Bumping the major or minor of an extension will require a manually uninstall/reinstall.

Reply:
I tested this after the release of the new version. I was offered the option to "Update" my extension to the latest version via a button in the dashboard. After clicking that button, the new scopes were added to the API permissions granted to my M2M client, as expected 👍

Reply:
Oh great! Thanks for testing that. I'll update our internal notes about this.